x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -0,0 +1,329 @@
+  ]);
+
+  const {
+    items,
+    totalItemCount,
+    tableOptions: { pageIndex, sort },
+  } = data;
+
+  const columns: Array<EuiBasicTableColumn<ServiceTransactionGroupItem>> = [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnName',
+        {
+          defaultMessage: 'Name',
+        }
+      ),
+      render: (_, { name, transaction_type: transactionType }) => {
+        return (
+          <TransactionGroupLinkWrapper>
+            <EuiToolTip delay="long" content={name}>
+              <StyledTransactionDetailLink
+                serviceName={serviceName}
+                transactionName={name}
+                transactionType={transactionType}
+              >
+                {name}
+              </StyledTransactionDetailLink>
+            </EuiToolTip>
+          </TransactionGroupLinkWrapper>
+        );
+      },
+    },
+    {
+      field: 'latency',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnLatency',
+        {
+          defaultMessage: 'Latency',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { latency }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis1"
+            compact
+            series={latency.timeseries ?? undefined}
+            valueLabel={
+              latency.value !== null
+                ? asDuration(latency.value)
+                : NOT_AVAILABLE_LABEL
+            }
+            start={parseFloat(start!)}
+            end={parseFloat(end!)}
+          />
+        );
+      },
+    },
+    {
+      field: 'traffic',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnTraffic',
+        {
+          defaultMessage: 'Traffic',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { traffic }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis0"
+            compact
+            series={traffic.timeseries ?? undefined}
+            valueLabel={
+              traffic.value !== null
+                ? asTransactionRate(traffic.value)
+                : NOT_AVAILABLE_LABEL
+            }
+            start={parseFloat(start!)}
+            end={parseFloat(end!)}
+          />
+        );
+      },
+    },
+    {
+      field: 'error_rate',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnErrorRate',
+        {
+          defaultMessage: 'Error rate',
+        }
+      ),
+      width: px(unit * 8),
+      render: (_, { error_rate: errorRate }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis7"
+            compact
+            series={errorRate.timeseries ?? undefined}
+            valueLabel={
+              errorRate.value !== null
+                ? asPercent(errorRate.value, 1)
+                : NOT_AVAILABLE_LABEL
+            }
```

Comment:
Similar to `asDuration` and `asTransactionRate` we might want to allow `Maybe<number>` for `asPercent` and handle the "n/a" state in `asPercent`:

```suggestion
            valueLabel={asPercent(errorRate.value, 1)}
```

```ts
export function asPercent(
  numerator: Maybe<number>,
  denominator: Maybe<number>
) {
  if (!isNumber(numerator) || !isNumber(denominator)) {
    return NOT_AVAILABLE_LABEL;
  }

 // ...
}
```